float32 kLST # linear stiffness
float32 kAST # angular stiffness
float32 kVST # volume stiffness
int32 flags # Default?
